spec/support/factory_girl.rb

```diff
@@ -1,10 +1,28 @@
 require 'factory_girl'
 
 FactoryGirl.define do
+  factory :game do
+    name "this is a game"
+    description "cards and stuff"
+    min_players 1
+    max_players 2
+    playing_time "20 min"
+    complexity "Low"
+      factory :game1 do
```

Comment:
What is the difference between `:game` and `:game1`? If multiple game objects have the same attributes but you need them to be unique you can use a sequence:

```ruby
factory :game do
  sequence(:name) { |n| "This is a game #{n}" }
  sequence(:description) { |n| "Description #{n}" }
  min_players 1
  max_players 4
  # ...
end
```